### DISCUSSION TITLE
How hard would it be to port this to Raspberry Pi?

### DISCUSSION THREAD

**Discussion by sinclairfr** [omersiar/esp-rfid#457] [Ideas]
Hi,

Don't get me wrong, I'm loving this project so much... My main gate at home has an official ESP-RFID Relay Board connected to it. 
However, ESP8266 has a major flaw: the wifi connectivity is very unstable and there is no RJ45 connector.

I'm contemplating porting this project over to a Raspberry Pi and I'm wondering how "hard" and tricky the whole process would be...?

Do any of you guys have any advice on this? Anyone caring to join me on this fork adventure?

Thanks =)

**Comment by matjack1** [comment]
hey @sinclairfr porting to RPi seems quite a significant endevour to me :(

If it's for RJ45 connector probably a porting to the ESP32 would be more feasible and possibly compatible with this project? It's still a significant task though. If it's only for the ethernet, maybe you can find a workaround?
